airflow/contrib/operators/dataproc_operator.py

```diff
@@ -928,3 +930,125 @@ def execute(self, context):
         job.set_job_name(self.job_name)
 
         hook.submit(hook.project_id, job.build(), self.region)
+
+
+class DataprocWorkflowTemplateInstantiateOperator(BaseOperator):
+    """
```

Comment:
Do you want to add any template_fields into your operator? maybe template_id, region?

Reply:
Added template_id